vpn/bytes_test.go

```diff
@@ -16,98 +19,366 @@ func Test_genRandomBytes(t *testing.T) {
 	}
 }
 
-func Test_encodeOptionString(t *testing.T) {
-
-	veryLargeStr := string(make([]byte, 1<<16))
-
+func Test_encodeOptionStringToBytes(t *testing.T) {
 	type args struct {
 		s string
 	}
-
 	tests := []struct {
 		name    string
 		args    args
 		want    []byte
-		wantErr bool
-	}{
-		{"goodEncode", args{"test"}, []byte{0, 5, 116, 101, 115, 116, 0}, false},
-		{"empty", args{""}, []byte{0, 1, 0}, false},
-		{"large", args{veryLargeStr}, nil, true},
+		wantErr error
```

Comment:
oh, I wonder why the `:GoTests` function I was using gave me bool in the template. will tweak my tooling, this is more precise...

Reply:
So, I manually rewrote them to a style that I like better. VSCode's equivalent of `:GoTests` generates a `bool` for me as well. I suppose this happens because `:GoTests` predates `errors.Is`.